### PR TITLE
Update quant_utils.py/write_calibration_table

### PR DESCRIPTION
**Pull request by aimilefth**
Adding a saving path allows to dictate where the calibration files will be saved.

### Description

Added a saving_path as an argument to the write_calibration_table function. That way, the files are not strictly saved on the currect running directory

### Motivation and Context

I run into an issue where I wanted to use the write_calibration_table and I was unable to open a file to the current directory, because of permission limitations.

Adding a saving path allows to dictate where the calibration files will be saved, therefore I can suggest a directory I know I have permissions

### CODE REVIEW

File: onnxruntime/python/tools/quantization/quant_utils.py
Comment (justinchuby) [inline onnxruntime/python/tools/quantization/quant_utils.py:473]: Just call it `dir`?

Comment (aimilefth) [reply onnxruntime/python/tools/quantization/quant_utils.py]: Renaming saving_path to dir

**Review by justinchuby — APPROVED** [review]
LGTM. Would be good to get a review from the code owners.

**Comment by hariharans29** [comment]
CC: @yufenglee

**Review by yufenglee — APPROVED** [review]

**Pull request merged**
